### DISCUSSION TITLE
Empty states documentation

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1754] [General] [answered]
The illustrations use are Twilio focused in the documentation. I'm wondering if there's plans to make the illustrations with color variations to adapt to other company's branding like SendGrid or Segment. Or if the plan is for everyone to shift towards Twilio branding
https://paste.twilio.design/patterns/empty-states/#whats-missing

**Comment by richbachman** [comment, marked as answer]
Hi @wilfredomerced. Good question. I'm checking in with the team, and will get back with you asap.

**Reply by gloriliale** [reply]
Hey @wilfredomerced! 

We looked into it and since the illustrations are owned by Brand, we actually cannot recolor them, but they might have better guidance on how/whether new SendGrid illustrations are being created. 

In the interim, if you need an empty state illustration specifically, our suggestion would be going w/ a large icon.
